umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/echo10/data_contact.clj

```diff
@@ -45,7 +46,9 @@
   "Science Software Development Manager" "Science Software Development"
   "Deputy Science Software Development Manager" "Science Software Development"
   "Sea Ice Algorithms" "Science Software Development"
-  "Snow Algorithms" "Science Software Development"})
+  "Snow Algorithms" "Science Software Development"
+  "Science Contact" "Science Contact"
+  "Science Software Development" "Science Software Development"})
```

Comment:
In the ticket, it asked "Also please make the ECHO 10 map values case insensitive."  I don't see any case insensitive code?

Reply:
Current Acceptance Criteria doesn't include that.
Ingest the attached echo 10 record.
Retrieve the same record using the umm-json specification.
Verify that the Data Contact named "National Snow and Ice Data Center Originator" has the role of ORIGINATOR
Verify that the Data Contact Contact Person has a role of Science Contact and the other has a role of Science Software Development
Ingest the retrieved umm-json record and retrieve it as an echo 10 format record. Verify that a contact exits with two contact persons. The contacts role is ORIGINATOR and one contact person has a JobPosition of Science Contact and the other has a JobPosition of
Science Software Development.